cg/meta/workflow/rnafusion.py

```diff
@@ -64,6 +64,19 @@ def get_deliverables_template_content() -> list[dict]:
             file_path=resources.RNAFUSION_BUNDLE_FILENAMES_PATH,
         )
 
+    @property
```

Comment:
Properties are associated with attributes. Did you mean it to be a static method? Or perhaps a constant?

Reply:
I'm not sure if this would be a correct use of the [property decorator](https://docs.python.org/3/library/functions.html#property). 

In any case you're not invoking it correctly in `write_nextflow_config` with `content=self.get_nextflow_config_content`. Has to be `NfAnalysisAPI.get_nextflow_config_content` since it's a class getter at the end

Reply:
I don't think so, RnafusionAnalysisAPI inherits from NfAnalysisAPI and I'm accessing an attribute from within so self.attribute is ok

Reply:
in any case, I can make it into a method if you prefer that. that's ok with me

Reply:
That's what I tried to point out in my first message—that you haven't defined `nextflow_config_content` as an attribute

Reply:
made into a method